app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -18,7 +18,20 @@ def index
 
       api :GET, '/job_invocations/:id', N_('Show job invocation')
       param :id, :identifier, :required => true
+      param :host_status, String, required: false, allow_blank: true, desc: N_('Show Job status for the hosts.')
       def show
+        @hosts = @job_invocation.targeting.hosts.authorized(:view_hosts, Host)
+        @template_invocations = @job_invocation.template_invocations
+                                               .where(host: @hosts)
+                                               .includes(:input_values)
+
+        if params[:host_status]
+          @template_invocations.includes(:run_host_job_task)
```

Comment:
This does not modify `@template_invocations` so this line essentially does nothing